simulators/swash/v09.01A/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM inductiva/kutu:base-image_v0.1.0 as test_env
+
+RUN apt-get update \
+    && apt-get install -y \
+    wget \
+    unzip &&\
+    wget https://storage.googleapis.com/inductiva-api-demo-files/swash-input-example.zip -P /home/ && \
+    unzip /home/swash-input-example.zip -d /home/ && \
+    rm /home/swash-input-example.zip
+
+COPY ./test_sim.sh /home/test_sim.sh
+RUN chmod +x /home/test_sim.sh
+
+FROM inductiva/kutu:base-image_v0.1.0 as build
+
+# Set SWASH options.
+ARG SWASH_VERSION=9.01A
+ARG SWASH_TMP=/tmp/swash
+
+# Download, build, and install SWASH.
+RUN mkdir ${SWASH_TMP}
+WORKDIR ${SWASH_TMP}
+
+RUN apt-get install ninja-build
+
+RUN git clone https://gitlab.tudelft.nl/citg/wavemodels/swash.git && \
+    cd swash && \
+    git checkout 9abb83229bcb5d04543a5ddafe707d34a25f39c0 && \
+    mkdir build && \
+    cd build && \
+    cmake .. -G Ninja -DMPI=ON -DCMAKE_Fortran_COMPILER=mpif90 -DCMAKE_INSTALL_PREFIX=/swash && \
+    cmake --build . && \
+    cmake --install . && \
+    cd / && \
+    rm -rf ${SWASH_TMP}
+
+
+FROM ubuntu:22.04
```

Comment:
Do we still need this one?

Reply:
> Do we still need this one?

Right now it is the best option. We would need to evaluate and test other options before making the change to another image. 
This new image has 96.98 MB which is not bad.

Reply:
But shouldn't this be inside our base image?

Reply:
> But shouldn't this be inside our base image?

Our base image is only used for building the bins. It is big because it has all dependencies the simulators might need to compile (openmpi, math libraries, compilers ...).

After compiling the simulators we no longer need all those extra files.